__snapshots__/packages/mcdoc/test-out/__fixture__/struct/simple.spec.js

```diff
@@ -47,7 +47,7 @@ exports['mcdoc __fixture__ struct/simple 1'] = {
                     "value": {
                       "kind": "literal",
                       "value": {
-                        "kind": "number",
+                        "kind": "double",
                         "value": 1.19
```

Comment:
is there any meaningful difference in this diff? does it matter that we parse release versions as a "double" rather than just some number?

we should probably parse it as a "semantic version" type instead, but that's prob better for a later PR

Reply:
It's parsed as string when it's `1.19.3`.

According to mcdoc spec, when not specifying a number suffix, it is parsed as either an int or a double, depending on its value.